src/main.py

```diff
@@ -30,8 +31,9 @@ def outline(year, lang):
 @app.route('/<lang>/<year>/contributors')
 @validate
 def contributors(year, lang):
-    # TODO: Get contributor data and pass into the template.
-    return render_template('%s/%s/contributors.html' % (lang, year), contributors={}, supported_languages=supported_languages)
+def contributors(year, lang):
```

Comment:
Accidentally copied this line. Should be removed.

Reply:
Oops! Thanks!